src/game-logic/reducers/sellItem.js

```diff
@@ -93,7 +93,10 @@ export const sellItem = (state, { id }, howMany = 1) => {
     itemsSold: newItemsSold,
   }
 
-  state = processLevelUp(state, oldLevel)
+  // this processLevelUp call actually doesn't have to happen because addExperience
+  // will call it and this results in double processing. although it doesn't cause
+  // any negative side effects, we could remove it.. eh?
+  //state = processLevelUp(state, oldLevel)
```

Comment:
I think it makes sense to drop this in favor of calling `processLevelUp` from `addExperience` like you're suggesting here, assuming it doesn't break anything. Good thinking! Let's go with it. :)